charmcraft.yaml

```diff
@@ -0,0 +1,12 @@
+type: charm
+bases:
+  - build-on:
+      - name: "ubuntu"
+        channel: "22.04"
+    run-on:
+      - name: "ubuntu"
+        channel: "22.04"
+parts:
+  charm:
+    prime: 
+      - files/.
```

Comment:
Can we get a final new line here please?

Reply:
Regarding GH workflows, that is intended for a future PR :) This one only focuses on adding the test to the old sidecar branch. Since it hasn't been maintained for a long time, I'll need to make various changes to it regarding workflows, linting, testing, syntax, etc...